.github/workflows/reusable-python-publish_package.yml

```diff
@@ -3,11 +3,11 @@ name: Publish condan package to Artifactory
 on:
   workflow_call:
     inputs:
-        package-manager:
-            description: 'Package manager to use (eg. "conda", "poetry", "hatch")'
+        package-managers:
+            description: 'Matrix of Package manager to use (eg. ["conda", "poetry", "hatch"])'
```

Comment:
**question:** remove hatch until it is implemented by the workflow?

Reply:
Yes, I preferred to remove it until it was implemented, since I don't have a package to test it with. But it's pretty much the same as with poetry.